modules/deploy/partials/requirements.adoc

```diff
@@ -220,16 +220,11 @@ ifdef::env-kubernetes[EKS defaults to the ext4 file system. Use XFS instead wher
 
 ifdef::env-kubernetes[AKS often defaults to the ext4 file system. Use XFS instead where possible.]
 
-- Memory optimized: Memory-optimized VM sizes offer a high memory-to-CPU ratio.
+- General purpose: General purpose VM sizes provide balanced CPU-to-memory ratio. Ideal for testing and development, small to medium databases, and low to medium traffic web servers.	
 
-** https://learn.microsoft.com/en-us/azure/virtual-machines/ebdsv5-ebsv5-series[Ebsv5 series (NVMe)^]
-** https://learn.microsoft.com/en-us/azure/virtual-machines/ebdsv5-ebsv5-series[Ebdsv5 series (NVMe)^]
-
-- Storage optimized: Storage-optimized virtual machine (VM) sizes offer high disk throughput and IO.
-
-** https://learn.microsoft.com/en-us/azure/virtual-machines/lsv2-series[Lsv2 series^]
-** https://learn.microsoft.com/en-us/azure/virtual-machines/lsv3-series[Lsv3 series^]
-** https://learn.microsoft.com/en-us/azure/virtual-machines/lasv3-series[Lasv3 series^]
+** https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/general-purpose/ddv5-series?tabs=sizebasic[Standard_D2d_v5^]
+** https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/general-purpose/ddv5-series?tabs=sizebasic[Standard_D4d_v5^]	
+** https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/general-purpose/ddv5-series?tabs=sizebasic[Standard_D32d_v5^]	
```

Comment:
Should we link out to the section that lists each instance by name? https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/general-purpose/ddv5-series?tabs=sizebasic#sizes-in-series I could go either way, not a blocker imo.